Installer/scripts/install_kexts.sh

```diff
@@ -23,6 +23,14 @@ do
         # VirtualSMC has to be second, or VSMC plugins
         # will not work
         i=1
+    #BlueToolFixup must be loaded after Polaris22Fixup or WEG, or stuck IOResourceMatch in Monterey b6.
+    # Might be bug or race conditions on BlueToolFixup Side
+    elif [ $kext == "Polaris22Fixup.kext" -a $at -gt 0 ]; then
+        i=2
+    elif [ $kext == "WhateverGreen.kext" -a $at -gt 0 ]; then
+        i=3
+    elif [ $kext == "BlueToolFixup.kext" -a $at -gt 0 ]; then
+        i=4
```

Comment:
This may not give the correct exact ordering because it depends on when the entry is seen. I think it should still be good relative to each other but need to think about it some more.

Reply:
Correct. 
Some kexts need to be set in absolute order (e.g. Lilu should always be No.1).
Some other kexts only needs to be set in relative order (in this case). 
There must be some better ways for implementation.

Reply:
https://github.com/osy/HaC-Mini/commit/9cbc3d0bede11ed875b7cb0cf9d48dae2768e6af Please use the new Priority feature.